ix_challenge_list_posts_by_rating/test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,77 @@
+require "test_helper"
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @post = Post.create(id: 667784312, title: "Post", upvotes: 1, downvotes: 1)
```

Comment:
porque criar os objetos informando o id?

Reply:
Pois, se não for informado o ID, o teste de criação irá falhar. Pois, os posts que não têm o ID informado seguirão a ordem do autoincrement. E quando for ocorrer o teste da criação, ele dirá que o ID 1 já existe

Reply:
os testes não podem guardar valores. deve haver uma limpeza na BD sempre que os testes correm e nunca ficarmos dependentes de valor fixos.

Reply:
Corrigido nos últimos commits